lib/standards/html-elms.js

```diff
@@ -674,7 +674,8 @@ const htmlElms = {
     noAriaAttrs: true
   },
   picture: {
-    contentTypes: ['embedded', 'phrasing', 'flow'],
+    // In HTML, this is considered "embedded". For axe-core we don't want this since its children will not be hidden
```

Comment:
This matches the comment style from the rest of the doc and makes it a bit more clear why axe-core does not want children to be hidden

```suggestion
    // Note: spec change (do not count as embedded), because browsers do not hide text inside the picture element
```